doc/source/development/contributing_environment.rst

```diff
@@ -87,14 +88,24 @@ Option 1: using mamba (recommended)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 * Install `mamba <https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html>`_
+  (the instructions may direct you to install `miniforge <https://github.com/conda-forge/miniforge>`_,
+  which is OK - it includes mamba)
 * Make sure your mamba is up to date (``mamba update mamba``)
+* Create and activate the ``pandas-dev`` mamba environment using the following commands:
 
 .. code-block:: none
 
-   # Create and activate the build environment
    mamba env create --file environment.yml
    mamba activate pandas-dev
 
+* Ensure any python packages without a version specified in the ``environment.yml`` file
```

Comment:
This is not entirely necessary given the instructions describe a fresh install.

Reply:
Thanks.  Yes, agreed.  I'll remove this - it keeps it cleaner.

Reply:
Done.